### ISSUE TITLE
when I add too many fields, I can't delete a field row

### ISSUE THREAD

**Issue by fodereas** [mitchcapper/KPEntryTemplates#80]
when I add too many fields, the form interface can't show more , there is no scroll bar! 
the interface cannot be resized.
So I can't drag a row to blank area to delete it.
![image](https://user-images.githubusercontent.com/25607187/166452630-8390ecba-4e9e-408a-b865-4e90fa8efd93.png)

**Comment by tgardner42** [comment]
One can "scroll" through the grid using the Up and Down arrow keys to find the desired row, but the lack of a scroll bar certainly negatively impacts the user experience here.

**Comment by mitchcapper** [comment]
I agree the lack of scrollbar is not intuitive.  The counter is I am really out of width.   Adding a scrollbar either cuts downs the width by say 15% and requires me to resize everything (or have a super ugly horizontal scrollbar) I think this was why I designed it the way it is originally.  Let me know thoughts.
